.yarn/patches/@metamask-network-controller-npm-23.5.0-56b87eea93.patch

```diff
@@ -1,19 +1,88 @@
 diff --git a/PATCH.txt b/PATCH.txt
```

Comment:
Introduced in https://github.com/MetaMask/metamask-extension/pull/33249

Reply:
Hmm, why write a new patch for 23.5.0 instead of updating to 23.5.1, as we did on `main`? https://github.com/MetaMask/metamask-extension/pull/33472

Reply:
Sorry coming back to u late on this one, please find more details here: https://consensys.slack.com/archives/C08UR8LL6BT/p1748966010988149?thread_ts=1748453321.090989&cid=C08UR8LL6BT

Reply:
I decided not to upgrade to 23.5.1 because from what I understood, we did not want to include a fix for the infinite loader issue in this release since it was not a release blocker. We were going to instead defer that fix to 12.20.1. So the patch just includes the `selectedNetworkClientId`. Does this match what you said @DDDDDanica?